server/app/views/admin/programs/ProgramBlocksView.java

```diff
@@ -1197,7 +1197,8 @@ private Modal renderBlockDescriptionModal(
         .withId("block-edit-form")
         .with(
             div(
-                    h1("The following fields will only be visible to administrators")
+                    h1("The screen name and description will help a user understand which part of"
```

Comment:
question (non-blocking): where does this text show up? I'm curious why it didn't generate a new screenshot.

Reply:
it's in a dialog that gets shown when you hit "edit screen name and description", which it seems like doesn't get clicked and screenshotted in any tests